spec.html

```diff
@@ -65,47 +65,53 @@ <h1>Import Calls</h1>
       <emu-clause id="sec-import-call-runtime-semantics-evaluation">
         <h1>Runtime Semantics: Evaluation</h1>
 
+        <emu-clause id="sec-evaluate-import-call">
+          <h1>EvaluateImportCall ( _specifierExpression_ [ , _optionsExpression_ ] )</h1>
+          <emu-alg>
+            1. Let _referencingScriptOrModule_ be ! GetActiveScriptOrModule().
+            1. Let _specifierRef_ be the result of evaluating _specifierExpression_.
+            1. Let _specifier_ be ? GetValue(_specifierRef_).
+            1. If _optionsExpression_ is present, then
+              1. Let _optionsRef_ be the result of evaluating _optionsExpression_.
+              1. Let _options_ be ? GetValue(_optionsRef_).
+            1. Else,
+              1. Let _options_ be *undefined*.
+            1. Let _promiseCapability_ be ! NewPromiseCapability(%Promise%).
+            1. Let _specifierString_ be ToString(_specifier_).
+            1. IfAbruptRejectPromise(_specifierString_, _promiseCapability_).
+            1. Let _assertions_ be a new empty List.
+            1. If _options_ is not *undefined*, then
+              1. Let _assertionsObj_ be Get(_options_, *"assert"*).
+              1. IfAbruptRejectPromise(_assertionsObj_, _promiseCapability_).
+              1. If _assertionsObj_ is not *undefined*,
+                1. If Type(_assertionsObj_) is not Object,
+                  1. Perform ! Call(_promiseCapability_.[[Reject]], *undefined*, &laquo; a newly created *TypeError* object &raquo;).
+                  1. Return _promiseCapability_.[[Promise]].
+                1. Let _keys_ be EnumerableOwnPropertyNames(_assertionsObj_, ~key~).
+                1. IfAbruptRejectPromise(_keys_, _promiseCapability_).
+                1. Let _supportedAssertions_ be ! HostGetSupportedImportAssertions().
+                1. For each String _key_ of _keys_,
+                  1. Let _value_ be Get(_assertionsObj_, _key_).
+                  1. IfAbruptRejectPromise(_value_, _promiseCapability_).
+                  1. If Type(_value_) is not String, then
+                    1. Perform ! Call(_promiseCapability_.[[Reject]], *undefined*, &laquo; a newly created *TypeError* object &raquo;).
+                    1. Return _promiseCapability_.[[Promise]].
+                  1. If _supportedAssertions_ contains _key_, then
+                    1. Append { [[Key]]: _key_, [[Value]]: _value_ } to _assertions_.
```

Comment:
```suggestion
                    1. Append { [[Key]]: _key_, [[Value]]: _value_ } to _assertions_.
              1. Sort _assertions_ by the code point order of the [[Key]] of each element. NOTE: This sorting is observable only in that hosts are prohibited from distinguishing among assertions by the order they occur in.
```
The sorting step after the loop was dropped; we should keep that.